talentlink-connector-product/product.json

```diff
@@ -22,14 +22,14 @@
         ]
       }
     },
-    {
-      "id": "maven-dependency",
+        {
+      "id": "maven-import",
       "data": {
-        "dependencies": [
+        "projects": [
           {
-            "groupId": "MY-GROUP-ID",
-            "artifactId": "MY-PRODUCT-NAME",
-            "version": "${version}",
+            "groupId": "com.axonivy.connector.talentlink.demo",
+            "artifactId": "talentlink-demo",
+            "version": "1.0.0",
```

Comment:
the version should remain dynamic by using the '${version}' keyword. The version to install will be provided at runtime through the market interface